companies/nielsen.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "nielsen",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "ads"
+    ],
+    "name": "Nielsen Holdings Plc",
+    "runs": [
+        "VisualDNA"
+    ],
+    "address": "ATTN: Legal Dept.\nNielsen\nNielsen House\nOxford Business Park South\nJohn Smith Drive\nOxford OX4 2WB\nUK",
+    "email": "privacy.department@nielsen.com",
```

Comment:
As that email goes directly to the privacy dept, we can set the suggested transport medium to `email`.